src/sentry/monitors/utils.py

```diff
@@ -39,6 +43,13 @@ def signal_first_monitor_created(project: Project, user, from_upsert: bool):
         )
 
 
+def signal_monitor_created(project: Project, user, from_upsert: bool):
+    cron_monitor_created.send_robust(
+        project=project, user=user, from_upsert=from_upsert, sender=Project
+    )
+    signal_first_monitor_created(project, user, from_upsert)
```

Comment:
do we need logic here to check if it's the first monitor in that project?

Reply:
ahh yea perhaps I can rename this function, but that method actually has the logic inside to check if its the first monitor

https://github.com/getsentry/sentry/pull/58017/files#diff-40b47f01f21b6be3039b55f92cd6cc51613fc6c0cce17837e5310cffc087069aR39

maybe a name like `check_and_signal_first_monitor_created` would be more fitting will change

Reply:
also the reason why I don't just put the logic inside this function, is because the `check_and_signal_first_monitor_created` is called in other places as well, so didn't want to duplicate the logic